File: wwaLabs/src/main/java/miu/edu/wwalabs/model/Post.java
package miu.edu.wwalabs.model;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.Cascade;

import java.util.ArrayList;
import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@Entity
public class Post {
    @Id
    @GeneratedValue
    long id;
    String title;
    String content;
    String author;
    @OneToMany(cascade = CascadeType.ALL)
    @JoinColumn(name = "post_id")
    private List<Comment> comments = new ArrayList<>();

}

File: WAALab7/src/Component/Post.js
import React, { useState } from 'react'
import PostDetails from './PostDetails'

function Post(props) {
 
  return (
    <div className="Content">
      <h3>ID: {props.id}</h3>
      <h3>Author: {props.author}</h3>
      <h3>Title: {props.title}</h3>
    </div>
  )
}

export default Post